File: docs/components_page/components/progress/multiple.R
library(dashBootstrapComponents)

progress <- dbcProgress(
  list(
    dbcProgress(valu = 20, color = "success", bar = TRUE),
    dbcProgress(value = 30, color = "warning", bar = TRUE),
    dbcProgress(value = 20, color = "danger", bar = TRUE)
  )
)

File: docs/components_page/components/progress/multiple.jl
using DashBootstrapComponents

progress = dbc_progress([
    dbc_progress(value = 20, color = "success", bar = true),
    dbc_progress(value = 30, color = "warning", bar = true),
    dbc_progress(value = 20, color = "danger", bar = true),
],);
